src/NodeApi.DotNetHost/NodeApi.DotNetHost.csproj

```diff
@@ -12,6 +12,14 @@
     <ProjectReference Include="..\NodeApi\NodeApi.csproj" GeneratePathProperty="true" PrivateAssets="all" />
   </ItemGroup>
 
+  <Target Name="PublishNativeHost" BeforeTargets="Pack">
```

Comment:
><Target Name="PublishNativeHost" BeforeTargets="Pack"> [](http://example.com/codeflow?start=2&length=54)

How it would work in the CI test I am working on?
Currently I publish in different environments and then collect the binaries to pack them in the single job. Binding publishing and packing makes it impossible to do. What is the new proposed algorithm?

Reply:
I added `Condition=" '$(NoPublish)' != 'true' "` here. So you can skip the auto-publish like this:
```
dotnet pack -p:NoPublish=true
```